src/objects/Term_Grade__c.object

```diff
@@ -0,0 +1,233 @@
+                    <default>false</default>
+                    <label>Dropped</label>
+                </value>
+                <value>
+                    <fullName>Fail</fullName>
+                    <default>false</default>
+                    <label>Fail</label>
+                </value>
+                <value>
+                    <fullName>Pass</fullName>
+                    <default>false</default>
+                    <label>Pass</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Term_Grade_Type__c</fullName>
+        <description>The type of term or grading period for which this grade is being recorded.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The type of term or grading period for which this grade is being recorded.</inlineHelpText>
+        <label>Term Grade Type</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Final Grade</fullName>
+                    <default>false</default>
+                    <label>Final Grade</label>
+                </value>
+                <value>
+                    <fullName>Grading Period</fullName>
+                    <default>false</default>
+                    <label>Grading Period</label>
+                </value>
+                <value>
+                    <fullName>Mid-Term Grade</fullName>
+                    <default>false</default>
+                    <label>Mid-Term Grade</label>
+                </value>
+                <value>
+                    <fullName>Exam Grade</fullName>
+                    <default>false</default>
+                    <label>Exam Grade</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Term__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The term or grading period for which this grade is being recorded.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The term or grading period for which this grade is being recorded.</inlineHelpText>
+        <label>Term</label>
+        <referenceTo>Term__c</referenceTo>
+        <relationshipLabel>Term Grades</relationshipLabel>
+        <relationshipName>Term_Grades</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <label>Term Grade</label>
+    <listViews>
+        <fullName>All</fullName>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+    </listViews>
+    <nameField>
+        <label>Term Grade Name</label>
```

Comment:
Per the specs, I expected this to be an autonumber and not have to specify a name.